python/ray/serve/_private/default_impl.py

```diff
@@ -38,6 +38,18 @@ def create_deployment_scheduler(
     )
 
 
+def create_dynamic_handle_options():
+    from ray.serve.handle import _DynamicHandleOptions
+
+    return _DynamicHandleOptions()
+
+
+def create_init_handle_options(**kwargs):
+    from ray.serve.handle import _InitHandleOptions
+
+    return _InitHandleOptions.create(**kwargs)
```

Comment:
why's this one have the **kwargs pattern but the dynamic one doesn't

Reply:
hmm, we could add **kwargs to make it more generalizable. The only place it's currently called is in the `DeploymentHandle` init method, where an empty `DynamicHandleOptions` is created - everywhere else dynamic options are created with `.copy_and_update()`.

Reply:
added **kwargs